--- qwtconfig.pri	2018-04-26 15:52:30.426530526 +0200
+++ qwtconfig_new.pri	2018-04-26 16:24:19.504809176 +0200
@@ -8,11 +8,13 @@
 VERSION      = $${VER_MAJ}.$${VER_MIN}.$${VER_PAT}

 unix {
-    INSTALLBASE    = /usr/local/qwt-5.2.0
+    #INSTALLBASE    = /usr/local/qwt-5.2.0
+     INSTALLBASE    = $$(PREFIX)
 }

 win32 {
-    INSTALLBASE    = C:/Qwt-5.2.0
+    #INSTALLBASE    = C:/Qwt-5.2.0
+     INSTALLBASE    = $$(LIBRARY_PREFIX)
 }

 target.path    = $$INSTALLBASE/lib